### PR TITLE
Initialized TripalTestSuite

### PR DESCRIPTION
**Pull request by carolyncaron**
I initialized [TripalTestSuite](https://github.com/statonlab/TripalTestSuite) for this module, so that I can start writing some unit tests! Please look over my setup :-)

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,3 @@
+gnore PHPUnit Test vendor scripts & environment info.
```
Comment (laceysanderson) [inline .gitignore]: I think this should be a comment: `# Gitignore PHPUnit Test vendor scripts & environment info.`

**Review by laceysanderson — APPROVED** [review]
Other then that, it looks good 👍

I'll approve despite the comment since that line will just be ignored as not matching anything so you won't see any problems if you leave it as is.

File: .gitignore (hunk 1 above)
Comment (carolyncaron) [reply .gitignore]: Good catch. Totally looks like I got bit by vim 😛

**Review by laceysanderson — APPROVED** [review]
Great, thanks!

**Pull request merged**